recipes/texture-atlas/meta.yaml

```diff
@@ -13,12 +13,12 @@ requirements:
   build:
     - cmake
     - ninja
-    - vtk
+    - vtk v8.1.1.*
```

Comment:
It looks like the difference between the conda-forge package and the kitware-geospatial package versions is the "v" prefix, since both are at 8.1.1. I think adding a comment to these lines would make that clear.